app/brave_settings_strings.grdp

```diff
@@ -476,6 +476,12 @@
   <message name="IDS_SETTINGS_BRAVE_SHIELDS_COOKIE_CONTROL_LABEL" desc="Default Brave cookies control setting label">
     Block cookies
   </message>
+  <message name="IDS_SETTINGS_BRAVE_SHIELDS_COOKIE_CONTROL_BLOCK_ALL_DEPRECATED_LABEL" desc="Block all cookies deprecation warning">
+    The "Block all Cookies" option has been deprecated. To learn how to force-enable this option, visit 
+  </message>
+  <message name="IDS_SETTINGS_BRAVE_SHIELDS_COOKIE_CONTROL_BLOCK_ALL_DEPRECATED_LINK_LABEL" desc="Block all cookies learn more link text">
+    our help page
+  </message>
```

Comment:
It is not great for localization when a sentence is split up between strings. It is better to use placeholders like you have in the Android file, but it should be all in one string (both here and in the Android file).